pkgs/development/python-modules/graph-tool/2.x.x.nix

```diff
@@ -15,15 +15,17 @@ stdenv.mkDerivation rec {
   };
 
   src = fetchurl {
-    url = "http://downloads.skewed.de/graph-tool/graph-tool-${version}.tar.bz2";
-    sha256 = "124qmd0mgam7hm87gscp3836ymhhwwnlfm2c5pzpml06da1w0xg9";
+    url = "https://github.com/count0/graph-tool/archive/release-${version}.tar.gz";
+    sha256 = "12w58djyx6nn00wixqnxnxby9ksabhzdkkvynl8b89parfvfbpwl";
   };
 
   preConfigure = ''
+    patchShebangs autogen.sh
+    ./autogen.sh
```

Comment:
No, was not aware about that.

But there seems to be no autoreconfHook in when im in the nix-shell. Where does it get defined?
I'm not familiar with this toolchain but autoreconf seems to be used for reconfiguring config files. Autogen needs to generate them first. In this version there is no config files in the archive.